packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.test.tsx

```diff
@@ -146,14 +153,17 @@ describe('PaymentMethodTitle', () => {
                 />,
             );
 
-            expect(component.find('[data-test="payment-method-logo"]').prop('src')).toBe(
+            const component = screen.getByRole('img');
+
+            expect(component?.getAttribute('src')).toEqual(
                 `${config.cdnPath}${LOGO_PATHS[method.id]}`,
             );
+            cleanup();
```

Comment:
While using `cleanup()` in each test ensures a clean testing environment, it may be redundant to repeatedly test the same scenario with the same type of data multiple times.

Can we get rid of both `forEach` and `clearnup` here?

Also, please replace `toEqual` here.

Reply:
Discussed in slack

Reply:
Hey @vitalii-koshovyi does it make sense to rewrite the test without using a for loop, for example
```
it('renders only logo for AmazonPay', () => {
    const methodIds = [
        PaymentMethodId.AmazonPay,
    ];

    render(
        <PaymentMethodTitleTest
            {...defaultProps}
            method={{
                ...defaultProps.method,
                methodIds,
            }}
        />,
    );

    const logos = screen.getByRole('img');
    const names = screen.queryByRole('heading');

    expect(logos).toBeInTheDocument();

    expect(names).not.toBeInTheDocument();
});

it('renders only logo for certain ChasePay', () => {
    const methodIds = [
        PaymentMethodId.ChasePay,
    ];

    render(
        <PaymentMethodTitleTest
            {...defaultProps}
            method={{
                ...defaultProps.method,
                methodIds,
            }}
        />,
    );

    const logos = screen.getByRole('img');
    const names = screen.queryByRole('heading');

    expect(logos).toBeInTheDocument();

    expect(names).not.toBeInTheDocument();
});
```

I believe its a small refactor and that way we can avoid using `cleanup` manually.